cmd/bffsrv/main.go

```diff
@@ -47,8 +49,12 @@ func getMode() (mode, error) {
 
 func main() {
 	log, _ := zap.NewProduction()
-	err := runE(log)
-	if err != nil {
+
+	if err := godotenv.Load(); err != nil {
+		log.Info("could not load .env file", zap.Error(err))
```

Comment:
It feels like this is going to be output every time the service starts in production - I wonder if there's a more natural way we can do this (e.g don't build this directly into the app, and instead get people to use https://webinstall.dev/dotenv/) ?